includes/templates/template_default/sideboxes/tpl_best_sellers.php

```diff
@@ -10,7 +10,7 @@
   $content = '';
   $content .= '<div id="' . str_replace('_', '-', $box_id . 'Content') . '" class="sideBoxContent">' . "\n";
   $content .= '<div class="wrapper">' . "\n" . '<ul class="list-links">' . "\n";
-  for ($i=1, $j=sizeof($bestsellers_list); $i<$j; $i++) {
+  for ($i=1, $j=sizeof($bestsellers_list); $i<=$j; $i++) {
```

Comment:
Changing the `for()` to `foreach()` would simplify all of this.  Simpler fix to the template, and no changes required on the module file.

Reply:
Many templates 'in the wild', Bootstrap included, _presume_ that the ordering of that array starts with `1`.

Reply:
Okay, then we can keep the proposed module changes. 
But this can still be changed to a foreach moving forward, and is probably the better advice to give to users, because it's the most-compatible "fix", both backward and forward.